docs/reference/migration/migrate_5_0/java.asciidoc

```diff
@@ -225,3 +225,58 @@ The `addSuggestion` method now required the user specified suggestion name, prev
 ===== SuggestionBuilder
 
 The `field` setter has been deleted. Instead the field name needs to be specified as constructor argument.
+
+==== SearchSourceBuilder
+
+All methods which take an `XContentBuilder`, `BytesReference` `Map<String, Object>` or `bytes[]` have been removed in favor of providing the
+relevant builder object for that feature (e.g. HighlightBuilder, AggregationBuilder, SuggesterBuilder) . This means that all search requests
+can now be validated at call time which results in much clearer errors.
+
+The `defaultResourceWindowSize(int)` method has been removed. The window size should be set explicitly on all RescoreBuilder objects.
```

Comment:
you may want to use the backtick for RescoreBuillder, same for HighlightBuilder, AggregationBuilder and SuggestBuilder wherever they are